docs/src/pages/components/data-grid/rendering/DensityPickerSmallGrid.js

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+import { useDemoData } from '@material-ui/x-grid-data-generator';
+
+export default function DensityPickerSmallGrid() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 4,
+    maxColumns: 6,
+  });
+
+  return (
+    <div style={{ height: 300, width: '100%' }}>
+      <DataGrid {...data} size="small" />
```

Comment:
at first sight, the prop name is bit weird and it does not feel linked to the density. Why not name it density?

Reply:
Already discussed in https://github.com/mui-org/material-ui-x/pull/606#discussion_r526992142.